src/core_plugins/testbed/public/testbed.html

```diff
@@ -0,0 +1,13 @@
+<div class="kuiTestbed kuiViewContent">
+  <div class="kuiViewContentItem">
+
+  <pre ng-if="testbed.data">{{ testbed.data }}</pre>
```

Comment:
This line isn't needed. It's a remnant of when we were testing the watchHistoryDetail PR.